docs/installing-solidity.rst

```diff
@@ -29,18 +29,20 @@ Further options on this page detail installing commandline Solidity compiler sof
 on your computer. Choose a commandline compiler if you are working on a larger contract
 or if you require more compilation options.
 
+.. _solcjs:
+
 npm / Node.js
 =============
 
 Use `npm` for a convenient and portable way to install `solcjs`, a Solidity compiler. The
 `solcjs` program has less features than all options further down this page. Our 
-`Using the compiler <using-the-compiler.html>` documentation assumes you are using
+:ref:`commandline-compiler` documentation assumes you are using
 the full-featured compiler, `solc`. So if you install `solcjs` from `npm` then you will
-stop reading the documentation here and then continue to <https://github.com/ethereum/solc-js>,
```

Comment:
This paragraph really has to be reworded at some point.

Reply:
I agree, but didn't had the motivation yet.